src/Sfa.Tl.ResultsAndCertification.Data/Repositories/AdminChangeLogRepository.cs

```diff
@@ -64,5 +65,76 @@ public async Task<PagedResponse<AdminSearchChangeLog>> SearchChangeLogsAsync(Adm
             List<AdminSearchChangeLog> changeLogs = await searchChangeLogQueryable.ToListAsync();
             return new PagedResponse<AdminSearchChangeLog> { Records = changeLogs, TotalRecords = totalCount, PagerInfo = pager };
         }
+
+        public async Task<AdminChangeLogRecord> GetChangeLogRecordAsync(int changeLogId)
+        {
+            IQueryable<ChangeLog> query =
+                _dbContext.ChangeLog
+                        .Include(p => p.TqRegistrationPathway)
+                            .ThenInclude(p => p.TqPathwayAssessments.Where(pa => pa.IsOptedin))
+                            .ThenInclude(p => p.TqPathwayResults.Where(pr => pr.IsOptedin))
+                            .ThenInclude(p => p.TlLookup)
+                        .Include(p => p.TqRegistrationPathway.TqPathwayAssessments.Where(pa => pa.IsOptedin))
+                        .ThenInclude(p => p.AssessmentSeries)
+                        .Include(p => p.TqRegistrationPathway.TqRegistrationSpecialisms.Where(rs => rs.IsOptedin))
+                            .ThenInclude(p => p.TqSpecialismAssessments.Where(sa => sa.IsOptedin))
+                            .ThenInclude(p => p.TqSpecialismResults)
+                            .ThenInclude(p => p.TlLookup)
+                        .Include(p => p.TqRegistrationPathway.TqRegistrationSpecialisms.Where(rs => rs.IsOptedin))
+                            .ThenInclude(p => p.TqSpecialismAssessments.Where(sa => sa.IsOptedin))
+                            .ThenInclude(p => p.AssessmentSeries)
+                        .Include(p => p.TqRegistrationPathway.TqRegistrationSpecialisms.Where(sa => sa.IsOptedin))
+                            .ThenInclude(p => p.TlSpecialism)
+                        .Include(p => p.TqRegistrationPathway.TqRegistrationProfile)
+                        .Include(p => p.TqRegistrationPathway.TqProvider)
+                            .ThenInclude(p => p.TqAwardingOrganisation)
+                            .ThenInclude(p => p.TlPathway)
+                        .Where(p => p.Id == changeLogId);
+
+
+            var profile = await query.Select(p => p.TqRegistrationPathway.TqRegistrationProfile).FirstOrDefaultAsync();
```

Comment:
All these select lines are not right. You have to run the query once and then you can read the inner properties